hack/aws-acceptance-test-cleanup/go.mod

```diff
@@ -1,9 +1,13 @@
 module github.com/hashicorp/consul-helm/hack/aws-acceptance-test-cleanup
 
-go 1.16
+go 1.18
 
 require (
 	github.com/aws/aws-sdk-go v1.38.63
 	github.com/cenkalti/backoff/v4 v4.1.1
+)
+
+require (
```

Comment:
why do we need two require blocks?

Reply:
https://github.com/hashicorp/consul-k8s/pull/878#discussion_r758834166